bin/license.py

```diff
@@ -102,14 +108,18 @@
     "Eclipse Public License": [],
     "Public Domain License": [],
     "Mozilla Public License Version 2.0": [],
-    "Unicode License": [],
+    "Unicode/ICU License": [],
     "Go License": [],
     "Other License": []
 }
 
 for _ in licenses:
+    # Because the license of this project itself dose not need to be declared here
     if "org.apache.seatunnel" in _:
         continue
+    # Because the license is not in compliance, if you need to use MySQL, you can add it yourself
+    if "mysql-connector-java" in _:
+        continue
```

Comment:
IMO, this is not needed

Reply:
You mean this line of code, or the MySQL dependence?

Reply:
We will not add mysql dependencies.

Reply:
But it has been added, seems you don’t know it

Reply:
About two years ago, this dependence was added by https://github.com/apache/incubator-seatunnel/commit/82297e4dfc61562b8f9a7984defaeb58b72a9465

Reply:
you can refer this https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L257

Reply:
FYI, https://github.com/apache/incubator-seatunnel/pull/1387#discussion_r818703096

Reply:
> Maybe it would better that you can merge this first and go through from the begin to understand the whole logic, and then we can discuss more effectively.

“Merge this first” is not a good practice IMO, codes get merged are rarely reviewed and re-discussed from my experience, we don’t have a deadline to deliver something in open source so let’s resolve it first before merging. 

The issue here to me is that you unnecessarily deal with the unused dependencies, because the plugin analyzed them, right?

did you try the option [`<excludedScopes>`](https://www.mojohaus.org/license-maven-plugin/add-third-party-mojo.html#excludedScopes) of that plugin to exclude test/provided scope dependencies, this is an easy-found option that can avoid this problem.

Reply:
Also because of this, I suspect there are more unnecessary dependencies are included which are provided/test scope

Reply:
@kezhenxu94 Cool, thanks big old 👍